projects/moodle-virtuallabx/moodle-backup.yaml

```diff
@@ -1,60 +1,58 @@
+---
 apiVersion: argoproj.io/v1alpha1
 kind: AppProject
 metadata:
-  name: moodle-backup
+  name: moodle-vlabx-backup
   namespace: argocd
 spec:
-  description: Project to backup uabierta databases
+  description: Project for moodle backup
```

Comment:
Sorry, no había cachado esto antes: mencionar VlabX en el description pues se ve en una vista que no tiene los demás detalles del proyecto a la vista. I.e.: "Project for backup of Virtuallabx's moodle"